inference-engine/tests/functional/inference_engine/core_threading_tests.cpp

```diff
@@ -136,14 +101,3 @@ TEST_F(CoreThreadingTests, DISABLED_GetAvailableDevices) {
     }, 30);
 }
 
-// tested function: ReadNetwork, AddExtension
-TEST_F(CoreThreadingTests, ReadNetwork) {
```

Comment:
Why did you remove this test?

Reply:
I moved it to inference-engine/tests/functional/inference_engine/extension/extension.cpp so I can exclude the tests if NGRAPH_ONNX_IMPORT_ENABLE==OFF or ENABLE_MKL_DNN=OFF

Reply:
this test is for `ReadNetwork`, not for extensions or CPU plugin. Don't understand why do we need to move it to another soruce.

Reply:
this test is for ReadNetwork and AddExtension. Extension that is used in this test case uses ONNX importer and MKLDNN plugin.

Reply:
I agree with @ilya-lavrenov.

This test didn't call load network and didn't depend on CPU Plugin.

Reply:
Yes, but it requires ONNX importer. That's why I moved this test to separate directory to exclude it when NGRAPH_ONNX_IMPORT_ENABLE==OFF. Other tests in extension.cpp require CPU plugin so that's why it's excluded when ENABLE_MKL_DNN=OFF.

Reply:
Yes I understand, but extension library doesn't depend on this flag: `ENABLE_MKL_DNN`. I think tests shouldn't depend on this flag if test doesn't use the CPU plugin.

Reply:
> That's why I moved this test to separate directory to exclude 

Why do we need to exclude something for ONNX case? Even for ONNX operators we have to be thread-safe. It does not matter whether extensions contains ONNX operator or not - that test must work